cvat-ui/src/components/annotation-page/standard3D-workspace/styles.scss

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+@import 'base.scss';
+
+.cvat-standard-workspace.ant-layout {
+    height: 100%;
+}
+
+.cvat-contextImage {
+    width: 250px;
+    position: absolute;
+    background: $border-color-3;
+    top: 10px;
+    right: 10px;
+    height: 150px;
+    z-index: 100;
+    border-radius: 6px;
+    //opacity: 0.5;
+    border: 1px solid $border-color-3;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 3px;
```

Comment:
For any static values lake this (margins, paddings, radius, etc) please use **$grid-unit-size** from base.scss. Also please use values which is multiples of 4, which is very important for our new design system. Like below:
`
padding: $grid-unit-size / 2;
border-radius: $grid-unit-size;
margin-top: $grid-unit-size * 2;
`

Reply:
I see this is completely copy/paste from standard workspace. Please make sure that you really need all these stuffs, and get rid of unnecessaries

Reply:
Resolved